ledger/src/shred.rs

```diff
@@ -1679,57 +1650,6 @@ mod tests {
         verify_shred_layout(&shred, &packet);
     }
 
-    #[test]
-    fn test_shred_flags() {
```

Comment:
This test only works with legacy shreds. Also if any of this logic fails all other tests that use shreds will fail.

Reply:
Looks like we have existing test coverage for all of these APIs
```
shred.data_complete()
shred.last_in_slot()
shred.reference_tick()
```

I'm okay axing this one